src/org/zaproxy/zap/extension/pscanrulesAlpha/ServerHeaderInfoLeakScanner.java

```diff
@@ -99,19 +98,25 @@ public int getPluginId() {
 	
 	@Override
 	public String getName(){
-		return Constant.messages.getString(MESSAGE_PREFIX + "name");
+		return Constant.messages.getString("pscanalpha.serverheader.scanner.name");

```

Comment:
Baseline scan (and friends) use the ID of the scanner, the name shown is for human consumption.